lib/imports.js

```diff
@@ -1,13 +1,65 @@
 import {
   createImportDeclaration,
   createNamedImportSpecifier,
-} from './createNode';
+} from "./createNode";
+
+const helpers = {
+  isExclusivelyDefaultImport: (importNode) =>
+    importNode.specifiers.length === 1 &&
+    importNode.specifiers[0].type === "ImportDefaultSpecifier",
+
+  /**
+   * Find an import specifier (import name) matching the moduleName in question.
+   * If a falsy moduleName argument is passed to the parent fn, this test will
+   * be ignored and will return true.
+   * @param   {Node}    importNode
+   * @returns {Boolean}
+   */
+  isMatchingName: (importNode, moduleName) => {
+    if (!moduleName) {
+      return true;
+    }
+
+    return isExclusivelyDefaultImport(importNode)
+      ? importNode.specifiers[0].local.name === moduleName
+      : importNode.specifiers &&
+          importNode.specifiers.find(
+            (specifier) =>
+              specifier.type === "ImportSpecifier" &&
+              specifier.imported &&
+              specifier.imported.name === moduleName
+          );
+  },
+
+  /**
+   * Find an import path matching the importPath in question.
+   * If no importPath argument is passed to the parent fn, this
+   * test will match against `@lendinvest/styleguide`.
+   * If importPath argument is set as `false` to the parent,
+   * this test will be ignored and will always return true.
+   * @TODO remove default to make it platform agnostic
+   * @param   {Node}    importNode
+   * @returns {Boolean}
+   */
+  isMatchingPath: (importNode, importPath) => {
+    if (importPath === false) {
+      return true;
+    }
+    return isExclusivelyDefaultImport(importNode)
+      ? importPath.test(importNode.specifiers[0].local.name)
```

Comment:
This looks like a mixture of a bug in original code and yours. This is not checking the import path, but imported item name?

but even if you changed it to what it is in code now:
```suggestion
      ? importPath.test(importNode.specifiers[0].source.value)
```
Then i still don't get how specifier can have source property.

Reply:
we don't need to access the specifier at all so `importNode.source.value`